packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -312,7 +312,7 @@ contract ForceMove is IForceMove {
         channelStorageHashes[channelId] = _hashChannelData(
             ChannelData(0, uint48(block.timestamp), bytes32(0), address(0), outcomeHash)
```

Comment:
An alternative would be to set `finalizesAt` to 0 during a conclude. Then it's a null value like most of the others and would not need to be emitted.

Reply:
... this would be a bad idea, though, since we use`0`as a magic value for `finalizesAt` indicating that a channel is open

https://github.com/statechannels/statechannels/blob/30f7226d6eac88eb28d1e5a4da4ce60842c70533/packages%2Fnitro-protocol%2Fcontracts%2FForceMove.sol#L735-L746